<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StatisticWindow</class>
 <widget class="QWidget" name="StatisticWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>235</width>
    <height>195</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Statistics</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>201</width>
     <height>141</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="expectedValueTextLabel">
        <property name="text">
         <string>Expected value is:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="expectedValueLabel">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="varianceTextLabel">
        <property name="text">
         <string>Variance is:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="varianceLabel">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="averageValueTextLabel">
        <property name="text">
         <string>Average value is:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="averageValueLabel">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
